mmv1/templates/terraform/examples/apigee_organization_keyvaluemaps_basic.tf.tmpl

```diff
@@ -0,0 +1,34 @@
+data "google_client_config" "current" {}
+
+resource "google_compute_network" "apigee_network" {
+  name       = "apigee-network"
+}
+
+resource "google_compute_global_address" "apigee_range" {
+  name          = "apigee-range"
+  purpose       = "VPC_PEERING"
+  address_type  = "INTERNAL"
+  prefix_length = 16
+  network       = google_compute_network.apigee_network.id
+}
+
+resource "google_service_networking_connection" "apigee_vpc_connection" {
+  network                 = google_compute_network.apigee_network.id
+  service                 = "servicenetworking.googleapis.com"
+  reserved_peering_ranges = [google_compute_global_address.apigee_range.name]
+}
+
+resource "google_apigee_organization" "apigee_org" {
+  analytics_region   = "us-central1"
+  project_id         = data.google_client_config.current.project
+  authorized_network = google_compute_network.apigee_network.id
+  depends_on         = [google_service_networking_connection.apigee_vpc_connection]
+}
+
+resource "google_apigee_organization_keyvaluemaps" "{{$.PrimaryResourceId}}" {
+  org_id    = google_apigee_organization.apigee_org.id
+  name      = "tf-test-org-kvms%{random_suffix}"
```

Comment:
1. There is an implicit dependency on `google_apigee_organization.apigee_org`, so you shouldn't need `depends_on` (this would apply to all of the examples I see)
2. This is docs-only, so name could probably be cleaner, like `kvms`

Reply:
dependency removed , thanks